docs/source/repositories.rst

```diff
@@ -0,0 +1,39 @@
+Repositories
+============
+
+Models and tokenizers can be loaded from repositories using the ``from_repo``
+method. You can add your own type of repository by implementing the
+:py:class:`curated_transformers.repository.Repository` base class.
```

Comment:
Having an example custom/toy implementation here would be nice for users.

Reply:
Great idea! I added a simple implementation.